OutOfSchool/OutOfSchool.WebApi/Controllers/V1/AboutPortalController.cs

```diff
@@ -17,22 +15,22 @@ namespace OutOfSchool.WebApi.Controllers.V1
     [Route("api/v{version:apiVersion}/[controller]/[action]")]
     public class AboutPortalController : ControllerBase
     {
-        private readonly IAboutPortalService informationAboutPortalService;
+        private readonly IAboutPortalService aboutPortalService;
         private readonly IStringLocalizer<SharedResource> localizer;
         private readonly ILogger<AboutPortalController> logger;
 
         /// <summary>
         /// Initializes a new instance of the <see cref="AboutPortalController"/> class.
         /// </summary>
-        /// <param name="informationAboutPortalService">Service for InformationAboutPortal model.</param>
+        /// <param name="aboutPortalService">Service for AboutPortal model.</param>
         /// <param name="localizer">Localizer.</param>
         /// <param name="logger"><see cref="Microsoft.Extensions.Logging.ILogger{T}"/> object.</param>
         public AboutPortalController(
-            IAboutPortalService informationAboutPortalService,
+            IAboutPortalService aboutPortalService,
             IStringLocalizer<SharedResource> localizer,
             ILogger<AboutPortalController> logger)
         {
-            this.informationAboutPortalService = informationAboutPortalService ?? throw new ArgumentNullException(nameof(informationAboutPortalService));
+            this.aboutPortalService = aboutPortalService ?? throw new ArgumentNullException(nameof(aboutPortalService));
```

Comment:
When Asp.Net Core tries to activate this controller, it will throw new InvalidOperationException if unable to resolve some services from DependencyInjection.ActivatorUtilities.GetService() in our cases.